charts/opentelemetry-operator/values.yaml

```diff
@@ -104,6 +104,14 @@ manager:
   env:
```

Comment:
@jtyr Minor request, can you please update the docstring here and recommend that people use extraEnvs instead of env please?

Reply:
I have added a short deprecation notice. Please review it.